docs/ui/insight/database-recommendations.md

```diff
@@ -0,0 +1,229 @@
+---
+title: Database recommendations
+linkTitle: Database recommendations
+description: Optimize usage of Redis databases
+weight: 40
+
+---
+
+RedisInsight provides database recommendations to enable you to use your databases more efficiently, leading to performance increase and memory usage optimization.
+
+## Avoid dynamic Lua script
+
+### Conditions
+
+Used_memory_lua > 3,000,000 (3M) bytes
+
+### Recommendations
```

Comment:
I need to thoroughly review this page with all the recommendations, but it definitely should be updated